src/components/gridlines.ts

```diff
@@ -67,10 +149,25 @@ namespace Plottable.Components {
       return this;
     }
 
+    private _mkTicks(scale: QuantitativeScale<any> | Plottable.Scales.Category): () => ScaleDomainValues {
+      if (QuantitativeScale.prototype.isPrototypeOf(scale)) {
+        return () => (<QuantitativeScale<any>>scale).ticks();
```

Comment:
consider an `.ticks` API for Category/Quantitative scale that you can call rather than the type check